src/main/java/app/gpx_animator/Configuration.java

```diff
@@ -489,15 +560,15 @@ public Builder photoTime(final Long photoTime) {
             return this;
         }
 
-        public Builder photoAnimationDuration(final Long photoAnimationDuration) {
-            this.photoAnimationDuration = photoAnimationDuration;
-            return this;
-        }
-
         public Builder addTrackConfiguration(final TrackConfiguration trackConfiguration) {
             this.trackConfigurationList.add(trackConfiguration);
             return this;
         }
+      
+      public Builder photoAnimationDuration(final Long photoAnimationDuration) {
+            this.photoAnimationDuration = photoAnimationDuration;
+            return this;
+        }
```

Comment:
If you move this method up above the `addTrackConfiguration` method, these changed lines will disappear and look like nothing changed here... 😊